dependencies/example-projects/grpc-vs-google-api-client/README.md

```diff
@@ -0,0 +1,62 @@
+# grpc 1.17.1 v.s. google-api-client 1.27.0
+
+This project is to demonstrate `NoSuchMethodError` generated from the class path
+for google-api-client and grpc-core.
+
+The package name `io.grpc.internal` is used for convenience, because DnsNameResolver
+is package private.
```

Comment:
Any way to make this happen without doing that by calling something higher up in the tree?

Reply:
Searching for the caller...

Reply:
I'm stuck at finding an example gRPC service that uses DNS TXT record for its configuration.
https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/internal/DnsNameResolver.java#L503

Reply:
As talked, I've added the explanation on why avoiding `io.grpc.internal` package is difficult.